parsec/core/cli/run.py

```diff
@@ -55,3 +57,121 @@ def run_mountpoint(config, device, mountpoint, timestamp, **kwargs):
         config = config.evolve(mountpoint_base_dir=Path(mountpoint))
     with cli_exception_handler(config.debug):
         trio_run(_run_mountpoint, config, device, timestamp)
+
+
+async def _import_file(workspace_fs, src, dest):
+    try:
+        await workspace_fs.touch(dest)
+    except FileExistsError:
+        await workspace_fs.truncate(dest, 0)
+    with open(src, "rb") as fd:
+        read_size = 0
+        while True:
+            chunk = fd.read(DEFAULT_BLOCK_SIZE)
+            if not chunk:
+                break
+            await workspace_fs.write_bytes(dest, chunk, read_size)
```

Comment:
> In fact I'm wondering if using workspace_fs.transactions.file_open is not even better for readability given fd_resize/fd_write are just as readable thantruncate /write_bytes, but we have a more fine control of what is done and the lock on the file from the open might offer interesting guarantees (@vxgmichel any opinion on this ?)

We had this plan of implementing a workspace async file object with an interface like those presented in the `io` module:
https://docs.python.org/3.8/library/io.html

It would be returned by `workspace_fs.open`:
```python
async with workspace_fs.file_open(fspath, "wb") as f:
    await f.write(some_binary_data)
```